xdp-dump/test-xdpdump.sh

```diff
@@ -377,6 +377,57 @@ test_none_xdp()
     fi
 }
 
+test_promiscuous()
+{
+   local PASS_PKT="packet size 118 bytes on if_name \"$NS\""
+   local PASS_REGEX="(@entry: packet size 118 bytes, captured 118 bytes on if_index [0-9]+, rx queue [0-9]+, id [0-9]+)"
```

Comment:
nit: indentation seems to be off here...

Reply:
Fixed in new commit.